dataplane/src/config.rs

```diff
@@ -0,0 +1,20 @@
+use std::collections::HashMap;
+
+pub struct Vpc {
+    #[allow(dead_code)]
+    pub vni: u32,
+}
+
+pub struct Config {
```

Comment:
Isn't it similar to https://github.com/githedgehog/dataplane/commit/b92a7b1f2e1bfe17d0943112cd524cae311d0b34#diff-c801271fc4b78850c43936f61ea0be4871f52b297f1be345b207f2a1a4aa5a92R201 << GlobalContext object that we store in NAT code?

Reply:
Yeah, the config here is just a dummy.  We need to agree on a common data model and put that in place of the placeholder I have here.